src/config.rs

```diff
@@ -64,6 +65,8 @@ impl Default for Options {
             lookahead_lines: 5,
             truncate_lines: true,
             ws_error_highlight: WsErrorHighlight::default(),
+            // TODO: decide on default
```

Comment:
I'm not sure if this is a viable solution since, as you mentioned there doesn't seem to be a reliable way to get the terminal's colors.

I think a "better" way could be creating a custom syntect theme and then map the `Style`s back to the 16 color ANSI sequenses and display them using crossterm. But I don't think I'll explore that option in this PR for now.

Either way, I agree that gex should be usable out-of-the-box even on terminals that don't support true-color / 24 bit colors. So yeah, the highlighting should be disabled by default.